typedarrays.go

```diff
@@ -630,6 +633,142 @@ func (a *float64Array) exportType() reflect.Type {
 	return typeFloat64Array
 }
 
+func (a *bigInt64Array) toRaw(value Value) uint64 {
+	return toBigInt64(value).Uint64()
+}
+
+func (a *bigInt64Array) ptr(idx int) *int64 {
+	p := unsafe.Pointer((*reflect.SliceHeader)(unsafe.Pointer(a)).Data)
+	return (*int64)(unsafe.Pointer(uintptr(p) + uintptr(idx)*8))
+}
+
+func (a *bigInt64Array) get(idx int) Value {
+	return valueBigInt{big.NewInt(*a.ptr(idx))}
+}
+
+func toBigInt64(v Value) *big.Int {
+	n := toBigInt(v).i
+
+	twoTo64 := new(big.Int).Lsh(big.NewInt(1), 64)
+	twoTo63 := new(big.Int).Lsh(big.NewInt(1), 63)
+
+	int64bit := new(big.Int).Mod(n, twoTo64)
+	if int64bit.Cmp(twoTo63) >= 0 {
+		return int64bit.Sub(int64bit, twoTo64)
+	}
+	return int64bit
+}
+
+func (a *bigInt64Array) set(idx int, value Value) {
+	*(a.ptr(idx)) = toBigInt64(value).Int64()
+}
+
+func (a *bigInt64Array) getRaw(idx int) uint64 {
+	return uint64(*a.ptr(idx))
+}
+
+func (a *bigInt64Array) setRaw(idx int, raw uint64) {
+	*(a.ptr(idx)) = int64(raw)
+}
+
+func (a *bigInt64Array) less(i, j int) bool {
+	return *(a.ptr(i)) < *(a.ptr(j))
+}
+
+func (a *bigInt64Array) swap(i, j int) {
+	pi, pj := a.ptr(i), a.ptr(j)
+	*pi, *pj = *pj, *pi
+}
+
+func (a *bigInt64Array) typeMatch(v Value) bool {
+	if _, ok := v.(valueBigInt); ok {
+		return true
+	}
+	return false
+}
+
+func (a *bigInt64Array) export(offset int, length int) interface{} {
+	var res []int64
+	sliceHeader := (*reflect.SliceHeader)(unsafe.Pointer(&res))
+	sliceHeader.Data = (*reflect.SliceHeader)(unsafe.Pointer(a)).Data + uintptr(offset)*8
+	sliceHeader.Len = length
+	sliceHeader.Cap = length
+	ret := make([]*big.Int, 0, length)
```

Comment:
I'm not sure it makes sense to export to an array of `big.Int`, considering the values are 64 bit and therefore would perfectly fit into `int64`. Same applies to BigUint64Array.